### ISSUE TITLE
Test on Edge browser

### ISSUE THREAD

**Issue by jeffbaumes** [girder/viime#437]
Seems to work ok on Chrome/FF.

I think Microsoft has some free VMs for testing out their browsers.

We may also want to see what happens on IE 11.

**Comment by subdavis** [comment]
I'll take a look at this today.

**Comment by subdavis** [comment]
* Download png doens't work anywhere.  No errors in console
* Download table works, but metabolite list and sample list do not.

**Comment by subdavis** [comment]
Up and down arrows do not appear next to these controls anywhere in the app

![Capture4](https://user-images.githubusercontent.com/4214172/68684680-e2394b80-0536-11ea-8853-2b6c5f248118.PNG)

**Comment by subdavis** [comment]
Select styling is wrong

![Capture7](https://user-images.githubusercontent.com/4214172/68684719-f54c1b80-0536-11ea-86fc-5be35dd629bb.PNG)

**Comment by subdavis** [comment]
PCA plot ellipse styling is broken

![Capture3](https://user-images.githubusercontent.com/4214172/68684740-ff6e1a00-0536-11ea-936d-bc41dd50308f.PNG)

**Comment by subdavis** [comment]
* Horizontal scroll bar in data validation issue navbar.  I started over and could not reproduce, so this is just something to look out for.

![Capture6](https://user-images.githubusercontent.com/4214172/68684807-16147100-0537-11ea-9868-5f2c25e5209d.PNG)

**Comment by subdavis** [comment]
Landing page scaling issues.  cc @jtomeck 

![Capture](https://user-images.githubusercontent.com/4214172/68685154-ad79c400-0537-11ea-866d-a9ec2355ed15.PNG)

**Comment by subdavis** [comment]
Vue infinite scroll table becomes confused.  This appears to be related to the uncommon browser behavior where scrolling "past" a page's end causes it to stretch past and snap back into alignment when you let go. I believe Safari does the same thing.  

I don't think we can really do anything about this.  you sort of have to "shake" the table by scrolling around fast to get into this state.

/shrug

![Capture2](https://user-images.githubusercontent.com/4214172/68685242-d13d0a00-0537-11ea-9720-9870d0929521.PNG)

**Comment by jeffbaumes** [comment]
With this many issues, @subdavis do you think it would be best to do browser detection and put a blocking warning on the front page if using IE/Edge saying it is not supported and to use Chrome/FF/Safari (we still need to check Safari)? Or @subdavis @sgratzl do you think these are resolvable in the next day or two?

We should fix the front page issues regardless if possible so I'd prioritize that.

**Comment by subdavis** [comment]
That's not an Edge issue after all.  It just happens on odd window sizes.  I'll take a stab at it.

![Screenshot from 2019-11-12 11-14-06](https://user-images.githubusercontent.com/4214172/68689030-c1282900-053d-11ea-99f2-15b23e52416b.png)

**Comment by subdavis** [comment]
@jeffbaumes yeah IMO a browser warning banner would be appropriate.  I don't have any apple hardware so someone else will need to check Safari.

**Comment by sgratzl** [comment]
several of the issues are just features edge is not capable of


> Up and down arrows do not appear next to these controls anywhere in the app

we use the native `input[type=number]` it depends on the browser how it renders (with arrows or not)

>     * PCA plot ellipse styling is broken

I assume Edge is not supporting 
https://www.w3.org/TR/SVGTiny12/painting.html#VectorEffectProperty

without it we have to change how to scale the ellipses (should be possible)


>     * Double scrollbars (this happens on every browser)

the outer is because of the left menu that needs more space. the inner because the plots have its own scrolling area to avoid that the header bar is scrolling. Thus kinda intentional

> Select styling is wrong

seems like Edge is not capable of the advanced gradient

**Comment by jeffbaumes** [comment]
@subdavis could you work on a banner of some sort?
